app/views/catalog/_index_masonry_default.html.erb

```diff
@@ -0,0 +1,2 @@
+<% #OVERRIDE blacklight-gallery: for shared search URLs %>
```

Comment:
Something looks off here. Syntax for line 1 comment, but line 2 looks like its commented, but I don't think it's supposed to be commented